wiki/Triaging-process.md

```diff
@@ -0,0 +1,59 @@
+
+        - Severity:
+            - **Severe**: Blocks or represents a primary user journey – e.g. end user playing a lesson, developer installing Oppia-Android.
+            - **Moderate**: blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log.
+            - **Minor**: Neither of the others, relates to “polish”.
+
+        ![image](https://user-images.githubusercontent.com/73544247/202834180-e26198bb-bc54-4fc9-9471-9348b439e5a2.png)
+
+    - ‘Work’ defines how hard it would be to solve/resolve a particular issue. This is generally correlated with the amount of ambiguity in the task. We use the following rough system for assigning this label:
+        - **Work: Low**: Solution is clear and broken into good-first-issue-sized chunks.
+        - **Work: Medium**: Otherwise, if the means to find the solution is clear.
+        - **Work: High**: Everything else.
+
+5. Once the appropriate ‘Impact’ and ‘Work Required’ labels has been added, determine its severity and assign the fitting priority label based on its impact (and possibly the work required label)
+    - If the issue is a high-impact bug, put it in the high-priority bucket and prioritise it within the team.
+    - For the priority bucket, team leads should include “important” issues – typically, a selection of high-impact enhancements and low-work medium-impact bugs – while keeping the size of that bucket about 2-3 times the projected stable number of team members.
+      > **Note**
+      > This exercise might also help determine how large the team needs to be.
+
+    - Other issues go in the ‘backlog’ bucket.
+    - Any issue with an external dependency that the team cannot resolve by itself (example: missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified in the ‘Issue specific status’ field (using free-text). Team leads should ensure an issue spends as little time here as possible.
+
+6. Determine if the issue is potentially a good first issue
+    - If the issue seems like something a new contributor to Oppia-Android can take up, make sure to add enough information in the issue description for a new contributor to start work on that issue then ask Ben or Adhiambo to add the Good First Issue label. (Note: typically these are “Work: Low” issues.)
+        - For such good first issues, also assign purple “technology: Kotlin”, “technology: Bazel”, “technology: Xml”, etc. labels as appropriate. This helps new contributors identify issues that they would find approachable.
+
+7. After triaging all issues, make sure the issues in the 'Priority' and 'High Priority' buckets are sorted based on severity/priority (high to low).
+
+
+## PM contacts for each team
+- **CLaM**:
+    - Lead: Ben Henning (@BenHenning)
```

Comment:
Actually I'm the PM (product manager) for this team.

Reply:
Done.